CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contribution Guidelines
```

Comment:
This file isn't referenced by `README.md`. Is it necessary, or is the one referenced below ([guidelines for contributing](https://github.com/esri/contributing)) sufficient?

The title for this section is "Contribution Guidelines", but then the section paragraph states _see our guidelines for contributing..._, which seems redundant.

I would be fine just referencing the main esri guidelines (https://github.com/esri/contributing).